src/applications/disability-benefits/996/config/form.js

```diff
@@ -40,6 +43,15 @@ const formConfig = {
   },
 
   formId: VA_FORM_IDS.FORM_20_0996,
+  saveInProgress: {
+    messages: {
+      inProgress:
+        'Your higher-level review application (20-0996) is in progress.',
+      expired:
+        'Your saved higher-level review application (20-0996) has expired. If you want to apply for higher-level, review please start a new application.',
+      saved: 'Your higher-level review application has been saved.',
```

Comment:
`Higher-Level Review` needs to be capitalized. And there shouldn't be a comma in between.

Reply:
The `Higher-Level Review` capitalization applies to the `expired` and `saved` content as well ;)

Reply:
Oh poop. I could have sworn... :man_facepalming:

Reply:
[Done!](https://github.com/department-of-veterans-affairs/vets-website/pull/14468/commits/501043323e449c680130f7984746810d7cf195d1)